tests/standalone/ClickUsing.py

```diff
@@ -0,0 +1,19 @@
+import click
+from click.testing import CliRunner
+
+# nuitka-skip-unless-imports: click, CliRunner
+
+def test_hello_world():
+	'''This function uses the CliRunner.invoke()
+	method which runs the command line script in
+	isolation and captures the output in both bytes and binary data'''
+	@click.command()
+	@click.argument('name')
+	def hello(name):
+		click.echo('Hello %s!'%name)
+
+	runner = CliRunner()
+	result = runner.invoke(hello,['Peter'])
```

Comment:
That is not interesting. I think @xennygrimmato told you already, we would like you to work on the `sys.argv` of the compiled program, making it use click methods to execute command line argument dependent behavior. The ``CliRunner` seems to have no point. We didn't compile it, so why bother.